catalog/task/validate-single-component/0.1/validate-single-component.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: validate-single-component
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task validates that the snapshot only contains a 
+    single component. A pipeline should fail otherwise.
+  params:
+    - name: snapshotPath
+      description: Path to the JSON string of the Snapshot spec in the data workspace
+      type: string
+      default: "snapshot_spec.json"
+  workspaces:
+    - name: data
+      description: Workspace where the snapshot is stored
+  steps:
+    - name: validate-single-component
+      image: quay.io/hacbs-release/release-utils:a918b3a7cbd40add5955a2f1acb4c3dbb29c44e9
+      script: |
+       #!/usr/bin/env sh
+       set -eux
+
+        # Obtain component from snapshot
+        COMPONENT_COUNT=$(jq '.components | length' $(workspaces.data.path)/$(params.snapshotPath))
```

Comment:
I would suggest putting the file path in quotes. It's not likely to happen, but this would break if there was a space in the workspace path or the filename. So: `"$(workspaces.data.path)/$(params.snapshotPath)"`

Reply:
Done, thanks!